include/term_translator.h

```diff
@@ -25,17 +25,93 @@
 #include "term.h"
 
 namespace smt {
+
+/** Class for translating terms from *one* other solver to *one* new solver
+ *  will fail if you try to convert terms from more than one solver
```

Comment:
can you give an example of this scenario? that would be nice to have.

Reply:
I added an example in the comments with e15b8eef97541ef9a279dca388d0aa58defaea59. Does that seem good?

Reply:
yeah, thanks